src/api/controllers/password.reset.js

```diff
@@ -14,48 +12,47 @@ const env = environment.currentEnv;
  */
 export default class PasswordReset {
   /**
-   * A controller to send an email to a user
-   * requesting password recovery
-   *
-   * @param {Object} req - the request object
-   * @param {Object} res - the result object
-   * @returns {Boolean} true
-   */
+	 * A controller to send an email to a user
+	 * requesting password recovery
+	 *
+	 * @param {Object} req - the request object
+	 * @param {Object} res - the result object
+	 * @returns {Boolean} true
+	 */
   static async sendRecoveryEmail(req, res) {
     // Initialising variables
     const result = {};
     const { username, email } = req.user;
 
     await mailer(`Password recovery for ${email}`, 'Password recovery', email, 'notifications', {
       email,
-      link: `${env.baseUrl}/api/password-reset`,
+      link: `${API_BASE_URL}/api/password-reset`,
```

Comment:
@murediane these changes are good, I know, but it makes it harder to review your PR. please, take out any changes to aren't related to this PR. We can have a separate PR later for improvements.